packages/ckb-testtool/src/unittest/core.ts

```diff
@@ -428,6 +434,65 @@ export class Resource {
     );
   }
 
+  /**
+   * Create a Script using ckb-js-vm.
+   * @param data - JS code path.
+   * @param args - The arguments to be used in the script.
+   * @param tx - The transaction to which the cell dependency will be added.
+   * @returns A Script object (With ckb-js-vm)
+   */
+  createJSScript(scriptPath: string, args: Hex,): Script {
```

Comment:
This function, with `completeTx` is different than original design. It's better to not add extra functions.